bundles/azure/aso/psql/bundle/config/01-flexible-server.ytt.yml

```diff
@@ -0,0 +1,234 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:regexp", "regexp")
+#@ load("@ytt:ip", "ip")
+
+#@ def tags(resource_tags=[]):
+#@  tags_list = []
+#@  tags_list.extend(data.values.global_tags)
+#@  tags_list.extend(resource_tags)
+#@  tags_map = { x["key"]: x["value"] for x in tags_list if x["key"] != None and x["value"] != None }
+#@  return(tags_map)
+#@ end
+
+#@ flexibleServerName    = data.values.name
+#@ dbSecretName          = data.values.name
+#@ resourceGroupK8SName  = data.values.name
+#@ serviceAccountName      = data.values.name + "-reader"
+#@ serviceAccountNamespace = data.values.namespace
+#@ roleName                = data.values.name + "-reading"
+
+#@ mandatory_fields = [ "name", "namespace", "location", "firewall_rules", "aso_controller_namespace" ]
+#@ missing_fields = []
+#@ for field in mandatory_fields:
+#@   if len(data.values[field]) == 0:
+#@     missing_fields.append(field)
+#@   end
+#@ end
+
+#@ databaseName = "name" in data.values.database and data.values.database.name or data.values.name
```

Comment:
_question_: Why not just

```starlark
#@ databaseName = data.values.database.name or data.values.name
```

Or is there something else you want to express here, which I don't get?

Reply:
Right, see comment above, a commit was missing.